### DISCUSSION TITLE
Welcome to node-red-contrib-socketcan Discussions!

### DISCUSSION THREAD

**Discussion by grodansparadis** [grodansparadis/node-red-contrib-socketcan#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by N3phyr** [comment]
Good afternoon :)

First of all I would like to thank you about this helpfull node. 

I'm a junior programmer from Spain, so excuse me about my english mistakes. I'm trying to use the socketcan to catch and notify heavy machine errors to prevent them

I was wondering if I could use it for multipacket messages. If no, do you have any advice about how to manage them?
